src/BTCPayServer.Lightning.Eclair/BTCPayServer.Lightning.Eclair.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\BTCPayServer.Lightning.Common\BTCPayServer.Lightning.Common.csproj" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <PackageReference Include="PureWebSockets" Version="2.4.3" />
```

Comment:
no dependency please, .NET already has websocket support.

Reply:
.net's websocket lib is quite shitty.. but done